### PR TITLE
docs: improve intro screenshots

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

File: docs/src/ci-intro-js.md [hunk 1]
```diff
@@ -92,16 +94,15 @@ Locally opening the report will not work as expected as you need a web server in
...
 
 To learn more about reports check out our detailed guide on [HTML Reporter](/test-reporters.md#html-reporter)
 
 ### Viewing the Trace
 
 Once you have served the report using `npx playwright show-report`, click on the trace icon next to the test's file name as seen in the image above. You can then view the trace of your tests and inspect each action to try to find out why the tests are failing.
 
-
-<img width="1907" alt="Viewing the Trace" src="https://user-images.githubusercontent.com/13063165/183879653-d442e6b4-14f5-4d0e-99f3-9ba19f82c7cf.png"/>
+<img width="2005" alt="Playwright Trace Viewer" src="https://user-images.githubusercontent.com/13063165/212746021-c3c5dabb-c5dd-4e4e-adf5-54b225d78dc8.png" />
```
Comment (mxschmitt) [inline docs/src/ci-intro-js.md:105]: % internal selector thing which should be fixed on next.

File: docs/src/running-tests-js.md [hunk 2]
```diff
@@ -79,28 +79,34 @@ Since Playwright runs in Node.js, you can debug it with your debugger of choice
...
 ```
 
-<img width="963" alt="HTML Report > Test Reports view" src="https://user-images.githubusercontent.com/13063165/189140120-71a6bc41-f921-40be-a7a4-61d44da33b20.png" />
+<img width="1392" alt="HTML Report > Test Reports view" src="https://user-images.githubusercontent.com/13063165/212744633-826cce1b-fab1-455a-8ca7-68867c4d4698.png" />
 
-You can click on each test and explore the tests errors as well as each step of the test.
+You can open a detailed view of each test by clicking on the test name. You can then explore the tests errors as well as expand each step of the test to see the code for that step and how long each step took to run.
 
-<img width="955" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/189140263-0e3ac2d9-4e75-40c8-b29c-58323c8be433.png" />
+<img width="1392" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/212744710-c48a327c-fb68-4c85-adca-2bdedbfcf504.png" />
```
Comment (mxschmitt) [inline docs/src/running-tests-js.md:109]: we should collapse one of them to show how detailed it is.

File: docs/src/ci-intro-js.md (hunk 1 above)
Comment (debs-obrien) [reply docs/src/ci-intro-js.md]: updated

File: docs/src/running-tests-js.md (hunk 2 above)
Comment (debs-obrien) [reply docs/src/running-tests-js.md]: done

File: docs/src/running-tests-js.md [hunk 3]
```diff
@@ -79,28 +79,41 @@ Since Playwright runs in Node.js, you can debug it with your debugger of choice
...
+<img width="1392" alt="HTML Report > Test Reports view" src="https://user-images.githubusercontent.com/13063165/212744633-826cce1b-fab1-455a-8ca7-68867c4d4698.png" />
+
+You can open a detailed view of each test by clicking on the test name. You can then explore the tests errors as well as expand each step of the test to see the code for that step and how long each step took to run.
+
+<img width="1247" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/212868173-2bf680bb-274a-4aec-932b-d07255adcc74.png" />
 
-You can click on each test and explore the tests errors as well as each step of the test.
 
-<img width="955" alt="HTML Reporter > Test Reports detailed view" src="https://user-images.githubusercontent.com/13063165/189140263-0e3ac2d9-4e75-40c8-b29c-58323c8be433.png" />
+For projects created with `create-playwright` the HTML report is enabled by default. If you have a config which does not use the HTML report or running the show-report command yields to no reports, you can run it with `--reporter=HTML`.
```
Comment (mxschmitt) [inline docs/src/running-tests-js.md:112]:
```suggestion
For projects created with `create-playwright` the HTML report is enabled by default. If you have a config which does not use the HTML report or running the show-report command yields to no reports, you can run it with `--reporter=html`.
```

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**